CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+## 1.0.0-beta
+
+**Date** - 8/19/2019
+
+**Release Tag** - [v1.0.0-beta](https://github.com/datacommonsorg/api-r/releases/tag/v1.0.0-beta)
+
+**Release Status** - Current head of branch [`1.0.0-beta`](https://github.com/datacommonsorg/api-r/tree/1.0.0-beta)
+
+Initial and beta release of the R API Client.
```

Comment:
Could you add a description here, mirroring the python version? (list of added functions, notebook, etc)

Reply:
Ah, ok. I mirrored the initial release note from Python. Is bbd94c3 better?

Reply:
yes, that looks better but it's not in this PR?

Reply:
resolved offline, ^ was a stale comment